django/src/rdwatch/serializers/hyper_parameters.py

```diff
@@ -10,6 +10,9 @@ class HyperParametersWriteSerializer(serializers.Serializer):
     performer = serializers.CharField()
     title = serializers.CharField(max_length=1000)
     parameters = serializers.JSONField(default=dict)
+    expiration_time = serializers.IntegerField(
+        min_value=0, max_value=24 * 365, required=False
```

Comment:
While I see that this has required=false, if you don't include it you get an error when posting models.